protostar/cheatable_starknet/callable_hint_locals/deploy_hint_local.py

```diff
@@ -21,12 +21,12 @@ def __init__(self, contracts_controller: ContractsController):
 
     @property
     def name(self) -> str:
-        return "deploy_tp"
+        return "deploy_impl"
 
     def _build(self) -> Callable[[list[int], int, int], Any]:
-        return self.deploy_prepared_tp
+        return self.deploy_prepared_impl
```

Comment:
```suggestion
        return self.deploy
```

Reply:
[Revert unwanted change, delete intermediate function and rename deplo…](https://github.com/software-mansion/protostar/pull/1845/commits/61d9b414d84f482849a114f029114024adb8c1ee)